guides/api.md

```diff
@@ -28,6 +30,26 @@ Each API Access Token specifies its `scopes`. Scopes allow a token to be used to
 
 Scopes, actions, and token types are expected to expand over time.
 
+## Write_Rooms
+The following parameters can be written to individual rooms on `createRoom` or `updateRoom`.
+- `name` _**str**_ \| The room name.
+- `description` _**str**_ \| A description of the room.
+- `roomSize` _**str**_ \| The number of non-admin participants allowed into the room from the lobby at any given time.
+- `sceneId` _**str**_ \| The seven character id of the scene hosted on the server.
+- `sceneUrl` _**str**_ \| A hosted asset to be used as the scene. Frequently used for specifying .glb files as scenes.
+- `memberPermissions` _**map**_ \| A map of the permissions non-admin participants should have in the room.
+  - `spawnAndMoveMedia` _**bool**_ \| Allow non-admin participants to spawn and move media.
+  - `spawnCamera` _**bool**_ \| Allow non-admin participants to spawn in-game cameras.
+  - `spawnDrawing` _**bool**_ \| Allow non-admin participants to draw with a pen.
+  - `pinObjects` _**bool**_ \| Allow non-admin participants to pin media to the room.
+  - `spawnEmoji` _**bool**_ \| Allow non-admin participants to spawn emoji.
+  - `fly` _**bool**_ \| Allow non-admin participants to toggle fly mode.
+  - `voiceChat` _**bool**_ \| Allow non-admin participants to use the voice chat.
+  - `textChat` _**bool**_ \| Allow non-admin participants to use the text chat.
+- `user_data` _**map**_ \| Arbitrary json data associated with this room.
+
+See [room_types.ex](../lib/ret_web/schema/room_types.ex) for full GraphQL Schema.
+
```

Comment:
Thank you for writing this. 

I am torn on whether to include it, because GraphQL has built-in `__schema` field that allows users to learn about a query: https://graphql.org/learn/introspection/

The built-in method has no risk of being outdated or wrong as we change things, whereas having this data duplicated here means that we risk forgetting to update it, or making a mistake.

I would not want to recreate the documentation that the graphql team already worked hard to write -- Perhaps we should link the graphql documentation and include some introspection queries in our sample workspace file.

Reply:
I did not know about this and agree we should utilize this tool which will prevent us from having to constantly update. I will review the docs tomorrow and put together a bit more of a step-by-step of how to quickly see what is available.